src/main/java/com/mattworzala/canary/server/givemeahome/Structure.java

```diff
@@ -1,16 +1,112 @@
 package com.mattworzala.canary.server.givemeahome;
 
+import com.google.gson.annotations.Expose;
 import net.minestom.server.coordinate.Vec;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.instance.block.BlockSetter;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 /**
  * This will be replaced eventually
- *
+ * <p>
  * todo(alex)
  */
 public class Structure {
 
+    @Expose
```

Comment:
Why are all of these `@Expose`? My understanding is its only relevant if you have fields which you do not want to serialize? Also in that case there is the java `transient` keyword.

Reply:
It was a case of I think i did two different things at once trying to solve a problem, and it turns out that one of them didn't actually solve it.